README.md

```diff
@@ -1,3 +1,31 @@
 # apollo-link
 
-This is the beginning of apollo-link. The new network interface layer for Apollo Client.
+`apollo-link` is a standard interface for fetching GraphQL results, designed to provide a simple GraphQL client that is capable of extensions.
+High level use cases are:
```

Comment:
This seems somewhat unfinished. I'd use more complete sentences, like "Here are the use cases where we think Apollo Link will be a good fit:"